post_office/mail.py

```diff
@@ -163,7 +164,9 @@ def send_many(kwargs_list):
     emails = []
     for kwargs in kwargs_list:
         emails.append(send(commit=False, **kwargs))
-    Email.objects.bulk_create(emails)
+    if len(emails) > 0:
+        Email.objects.bulk_create(emails)
+        email_queued.send(sender=Email, emails=emails)
```

Comment:
Is passing `emails=emails` here ok? The signal definition only mentions `email` as the argument.